pynumdiff/tests/test_optimize.py

```diff
@@ -25,57 +25,13 @@ def test_finite_difference():
     assert params1['num_iterations'] == 5
     assert params2['num_iterations'] == 1
 
-def test_mediandiff():
```

Comment:
It's silly to run the optimizer this many times on this many different functions. Nixed a few.